experiments.py

```diff
@@ -195,4 +195,110 @@ def run(self):
 
         model_fp = os.path.join(self.ph['log_dir'], self.ph['model_fp'].format(self.curr_run_number))
         model.save_weights(model_fp)
-        return
\ No newline at end of file
+        return
+
+class ConvDropoutSupervisedLearningFL(Algorithm):
```

Comment:
I might be missing something but how is this different from the SupervisedLearningFL algorithm? If you want to define a conv model with dropout, i think that would only require adding a ConvModel class to models/